app/controllers/RegisterTrustController.scala

```diff
@@ -82,7 +82,10 @@ class RegisterTrustController @Inject()(desService: DesService, config: AppConfi
                         response = response
                       )
 
-                      rosmPatternService.enrolAndLogResult(response.trn, request.affinityGroup) map {
+                      rosmPatternService.enrolAndLogResult(response.trn,
+                        request.affinityGroup,
+                        trustsRegistrationRequest.trust.details.trustTaxable.getOrElse(false)
```

Comment:
Wouldn't this be .getOrElse(true)?

It won't be present for current trusts being registered in 4MLD therefore we need to set taxable to true?

Reply:
Done